src/app/about/components/aboutSectionPositionList.jsx

```diff
@@ -6,16 +6,16 @@
  * @constructor
  */
 export default function AboutSectionPositionList({
-  header = "",
-  listText = [""],
-}) {
+                                                   header = "",
+                                                   listText = [""]
+                                                 }) {
   return (
     <div
       className={`w-[48%] max-lg:ml-[6%] min-w-[400px] max-lg:w-full max-sm:min-w-0 positions-container mb-4`}
     >
       <div className={`position-header text-xl/6 mb-2`}>{header}</div>
       <hr className={`w-[90%] border-0 h-px bg-(--neutral-200)`} />
-      <ul className={`positions-list list-disc mt-4`}>
+      <ul className={`positions-list list-disc mt-4 ml-6`}>
```

Comment:
- Remove the unused class `positions-list`.
- Add `list-inside`.
- Margin left should be 2 or 3.

```suggestion
      <ul className={`list-inside list-disc mt-4 ml-3`}>
```